library/src/main/scala/scala/scalajs/js/Array.scala

```diff
@@ -186,4 +187,8 @@ object Array {
   /** Returns true if the given value is an array. */
   @inline
   def isArray(arg: scala.Any): Boolean = NativeArray.isArray(arg)
+
+  /** Creates a new array from js.Iterable */
```

Comment:
This Scaladoc should have the ECMAScript 2015 badge:
```
<span class="badge badge-ecma2015" style="float: right;">ECMAScript 2015</span>
```

Reply:
Added badge and squashed commits.